### PR TITLE
Release ARM/ARM64 Nuget packages

### PR DESCRIPTION
**Pull request by tiagoshibata**
**Description**: Add ARM packages

### CODE REVIEW

File: csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj [hunk 1]
```diff
@@ -162,7 +162,7 @@
           Visible="false"
     />
     <!-- Some tools to be packaged in nightly build only, should not be released -->
-    <!-- These are copied to the runtimes folder for coveniennce of loading with the dlls -->
+    <!-- These are copied to the runtimes folder for covenience of loading with the dlls -->
```
Comment (orilevari) [inline csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj]: Nit: convenience

File: tools/ci_build/github/azure-pipelines/templates/win-ci-2019.yml [hunk 2]
```diff
@@ -72,6 +72,11 @@ jobs:
         arguments: '$(BuildCommand) --update --config RelWithDebInfo'
         workingDirectory: '$(Build.BinariesDirectory)'
 
+    - ${{ if notIn(parameters['sln_platform'], 'Win32', 'x64') }}:
+      # Use cross-compiled protoc
+      - script: |
+         @echo ##vso[task.setvariable variable=ProtocDirectory]$(Build.BinariesDirectory)\host_protoc\Release
```
Comment (zhangxiang1993) [inline tools/ci_build/github/azure-pipelines/templates/win-ci-2019.yml]: Why do you need to do this?

Comment (tiagoshibata) [reply tools/ci_build/github/azure-pipelines/templates/win-ci-2019.yml]: It's built here: https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/build.py#L783

When cross-compiling, protoc must be in the host architecture. It's used as part of the build system to compile the protobuf files.

I reference it here: https://github.com/microsoft/onnxruntime/pull/2987/files#diff-2a44500e30cc605574f7ac04ba287752R10

**Comment by snnn** [comment]
Hi @tiagoshibata, I feel the work was half-done. You created the new build jobs, but you didn't really put them in the final nuget package's runtime dir.  So the latest microsoft.ml.onnxruntime nuget package still doesn't have arm.

**Pull request merged**